### PR TITLE
restart simulated streamed request on stream close

### PR DESCRIPTION
**Pull request by phryneas**
This would solve the situation where the server renders a `useBackgroundQuery`, but does not consume the `queryRef`, so the stream closes before the result can be transported over.

### CODE REVIEW

File: package/src/ssr/ApolloNextAppProvider.tsx [hunk 1]
```diff
@@ -43,6 +43,8 @@ export const ApolloNextAppProvider = ({
   return (
     <_ApolloProvider
       client={clientRef.current}
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```
Comment (phryneas) [inline package/src/ssr/ApolloNextAppProvider.tsx:47]: This is just to get the CI to shut up until #61 is merged - I'll remove that later.

File: package/src/ssr/NextSSRApolloClient.tsx [hunk 2]
```diff
@@ -146,12 +153,44 @@ export class NextSSRApolloClient<
...
+              const queryId = queryManager.generateQueryId();
+              queryManager
+                .fetchQuery(queryId, {
+                  ...queryInfo.options,
+                  context: {
+                    ...queryInfo.options.context,
+                    queryDeduplication: false,
+                  },
+                })
+                .finally(() => queryManager.stopQuery(queryId))
```
Comment (phryneas) [inline package/src/ssr/NextSSRApolloClient.tsx:177]: See https://github.com/apollographql/apollo-client/blob/cdb98ae082ae4c7da6cd6a0fd5ad8457810fceda/src/core/QueryManager.ts

**Review by jerelmiller — APPROVED** [review]
🎉

**Review by alessbell — APPROVED** [review]
🎉

**Comment by phryneas** [comment]
Released in 0.4.1 https://github.com/apollographql/apollo-client-nextjs/releases/tag/v.0.4.1

**Pull request merged**